maas-api.yaml

```diff
@@ -1110,1966 +1120,1834 @@ paths:
+          popularity:
+            type: integer
+            description: Number for popularity of stop defined by number of services and their types.
+          modeInfo:
+            $ref: '#/components/schemas/ModeInfo'
+          wheelchairAccessible:
+            type: boolean
+            description: Is this stop wheelchair accessible? Missing when unknown.
         required:
-          - lat
-          - lng
-      components:
-        description: Components of this vehicle with additional information. The top level array represents connected parts of the vehicle, which you can't walk through without leaving the vehicle (e.g., two trains connected together). The inner level array represents then parts that can be walked through (e.g., the waggons of a train). A bus would have a `[[component1]]`. A train could have `[[c1, c2, c3, c4], [c5, c6, c7, c8]]`. The arrays are ordered by direction of travel always being left-to-right, i.e., the front of the train is the very last element.
-        type: array
-        items:
+          - code
+          - name
+          - popularity
+          - modeInfo
+
+    StopLocationParent:
+      allOf:
+      - $ref: '#/components/schemas/StopLocation'
+      - type: object
+        properties:
+          children:
+            type: array
+            items:
+              allOf:
+                - $ref: '#/components/schemas/StopLocation'
+                - type: object
+                  properties:
+                    childName:
+                      type: string
+                      description: Platform number
+
+    BikePodInfo:
+      properties:
+        identifier:
+          type: string
+        operator:
+          $ref: '#/components/schemas/CompanyInfo'
+        inService:
+          description: Whether this pod is currently in service (e.g., some might be closed for maintenance or during winter months)
+          type: boolean
+        totalSpaces:
+          description: Static information of total spaces
+          type: integer
+        availableBikes:
+          description: |
+            Real-time information of available bikes.
+
+            Note that this can be *more* than the total spaces for providers that have free-floating bicycles that are left at a station even though there's no bike dock.
+          type: integer
+        lastUpdated:
+          type: string
+          format: date-time
+          description: Timestamp `availableBikes` got recorded in ISO8601 format
+        source:
+          $ref: '#/components/schemas/DataSourceAttribution'
+      required:
+        - identifier
+        - operator
+
+    BikePodLocation:
+      allOf:
+      - $ref: '#/components/schemas/Location'
+      - type: object
+        properties:
+          bikePod:
+            $ref: '#/components/schemas/BikePodInfo'
+          modeInfo:
+            $ref: '#/components/schemas/ModeInfo'
+        required:
+          - bikePod
+          - modeInfo
+
+    CarParkInfo:
```

Comment:
you could use the updated car park schema from https://github.com/skedgo/TSP-APIs/pull/3/files